portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,13 +32,15 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that handles comments data */
-@WebServlet("/comments")
+@WebServlet("/get-comments")
 public class DataServlet extends HttpServlet {
 
+  private int maxComments = 1;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = 1;
     String maxCommentsString = getRequestParam(request, "max-comments");
+    System.out.println("maxComments is " + maxComments);
     
     if (!maxCommentsString.equals("")) {
       maxComments = Integer.parseInt(maxCommentsString);
```

Comment:
Generally it's a good idea to handle errors, even if the front end is doing some input validation/restriction for you. For example, what if maxCommentsString was "asdf" and wouldn't parse as an int?

Again since this is a simple learning project, this is probably fine though.

Reply:
Got it, I'll look out to add error checking in the future